.prettierignore

```diff
@@ -1,3 +1,3 @@
-.github/ISSUE_TEMPLATE
-.github/ISSUE_TEMPLATE.md
-.github/PULL_REQUEST_TEMPLATE.md
```

Comment:
We can undo this if there is a reason prettier shouldn't touch these files :)

Reply:
probably the intentional extra lines that I added, which prettier removed

Reply:
cool will being these back 👍